docs/src/userguide/language_basics.rst

```diff
@@ -381,6 +381,48 @@ Sometimes Cython will complain unnecessarily, and sometimes it will fail to
 detect a problem that exists. Ultimately, you need to understand the issue and
 be careful what you do.
 
+
+Type Casting
+------------
+
+As C uses the ``"("`` and ``")"`` for casting types, Cython uses ``"<"`` and ``">"``
+to signify a casting. For example::
```

Comment:
"to signify a casting" - I'd remove these words entirely. Just: "where C uses abc, Cythoin uses xyz."
Problem is that "a casting" reads badly and is probably not proper English.